src/components/dateTimePicker.js

```diff
@@ -0,0 +1,133 @@
+(() => ({
+  name: 'DateTimePicker',
+  icon: 'DateTimePickerIcon',
+  category: 'FORM',
+  type: 'CONTENT_COMPONENT',
+  allowedTypes: [],
+  orientation: 'HORIZONTAL',
+  jsx: (() => {
+    const {
+      label,
+      defaultValue,
+      required,
+      disabled,
+      error,
+      placeholder,
```

Comment:
I think we can remove the placeholder for this component? 

Don't have a way of showing it I think

Reply:
Yes you can, by emptying the value manually. 
![image](https://user-images.githubusercontent.com/11158471/77439775-c2206f80-6de7-11ea-939e-1db7d4a2c0a4.png)